packages/p/parchment.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "parchment",
+  "description": "A document model for rich text editors",
+  "keywords": [],
+  "author": {
+    "name": "Jason Chen",
+    "email": "jhchen7@gmail.com"
+  },
+  "license": "BSD-3-Clause",
+  "homepage": "http://quilljs.com/docs/parchment",
```

Comment:
the bot could warn for non https

Reply:
Warning would be useful, I don't think it should block the PR being approved though.